docs/en/observability/synthetics-command-reference.asciidoc

```diff
@@ -0,0 +1,53 @@
+[[command-line-options]]
+= Synthetics command reference
+
+++++
+<titleabbrev>Command reference</titleabbrev>
+++++
+
+experimental::[]
+
+[discrete]
+== `npx @elastic/synthetics` command
```

Comment:
this would be become as we cannot have scoped CLI bin names`npx synthetics`

Reply:
I'm not sure I understand